scripts/fiber/tests-passing.txt

```diff
@@ -1991,10 +1992,6 @@ src/renderers/testing/__tests__/ReactTestRenderer-test.js
 * can update text nodes when rendered as root
 * can render and update root fragments
 
-src/shared/utils/__tests__/KeyEscapeUtils-test.js
-* should properly escape and wrap user defined keys
-* should properly unescape and unwrap user defined keys
```

Comment:
I didn't bother porting this test, for isomorphic (which is where this code leaves on) it's only testing `flatten` which is not even officially exposed. But I can write it against `flatten` specifically if somebody cares.